### PR TITLE
feat(behavior_path_planner): output debug marker from each module

### PR DESCRIPTION
**Pull request by satoshi-ota**
## Description

- output debug marker from each module

image

```sh
➜  pilot-auto git:(awf-latest) ✗ ros2 topic list | grep behavior_path
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance_left/cooperate_status
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance_right/cooperate_status
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/debug/avoidance
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/debug/avoidance_debug_message_array
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/debug/lanechange
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/debug/lanefollowing
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/debug/pullout
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/debug/pullover
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/debug/sideshift
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/drivable_area_boundary
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/input/lateral_offset
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/lane_change_left/cooperate_status
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/lane_change_right/cooperate_status
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/output/path_candidate
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/pull_out/cooperate_status
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/pull_over/cooperate_status
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/pull_over/debug/Cl
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/pull_over/debug/Cr
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/pull_over/debug/parking_area
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/pull_over/debug/path_pose_array
/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/pull_over/debug/start_pose
```

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: planning/behavior_path_planner/src/behavior_path_planner_node.cpp [hunk 1]
```diff
@@ -596,7 +593,6 @@ void BehaviorPathPlannerNode::run()
 
   // for debug
   debug_avoidance_msg_array_publisher_->publish(bt_manager_->getAvoidanceDebugMsgArray());
-  publishDebugMarker(bt_manager_->getDebugMarkers());
```
Comment (kosuke55) [inline planning/behavior_path_planner/src/behavior_path_planner_node.cpp:599]: `getDebugMarkers()` function is no longer needed?

Comment (satoshi-ota) [reply planning/behavior_path_planner/src/behavior_path_planner_node.cpp:599]: I'll remove it :+1:

Comment (satoshi-ota) [reply planning/behavior_path_planner/src/behavior_path_planner_node.cpp:599]: Remove in https://github.com/autowarefoundation/autoware.universe/pull/1711/commits/8f72d46025e7a041eb772b239ad2e9c9aa5a0e30.

**Review by kosuke55 — APPROVED** [review]

**Comment by satoshi-ota** [comment]
TODO: I'll fix rviz configuraion.

**Comment by kosuke55** [comment]
update pull_over debug in https://github.com/autowarefoundation/autoware.universe/pull/1843

**Pull request merged**